src/components/Input/NormalInput.tsx

```diff
@@ -61,7 +64,7 @@ export default function NormalInput({ value, onChange, errorMsg, ...props }: Nor
           {errorMsg || props.description}
         </span>
 
-        {props.maxLength && (
+        {isMaxLengthTextVisible && (
           <span className={css(inputLengthWrapperCss, { color: statusColor })}>
```

Comment:
```suggestion
          <span className={cx(inputLengthWrapperCss,css({ color: statusColor }))}>
```
이런식으로 사용할 수 있게 변경하면 좋을 것 같아요

Reply:
[f7e5c88](https://github.com/depromeet/10mm-client-web/pull/301/commits/f7e5c88a6f7af8dacbe06d3aeff3bdd4562ae7e9) 여기서 변경해주었어요